### ISSUE TITLE
SC2005 and SC2116 overlap, giving two warnings for the same issue

### ISSUE THREAD

**Issue by arth1** [koalaman/shellcheck#218]
I have the following:

```
# Space separated list of network alias names
ALIASES=$(echo $(grep -v '^#' /etc/sysconfig/network-aliases 2>/dev/null))
        ^-- SC2116: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'.
               ^-- SC2046: Quote this to prevent word splitting.
               ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
```

Even if shellcheck doesn't know that echo sometimes is used to strip whitespace, it surely shouldn't report two warnings about useless echo for a single echo?

Is there any way to get the same result as the above without using three # shellcheck disable=  statements?  It seems overkill for a single short line...
